src/commands/scan/cmd-scan-reach.mts

```diff
@@ -50,6 +50,10 @@ const config: CliCommandConfig = {
     Usage
       $ ${command} [options] [CWD=.]
 
+    API Token Requirements
+      - Quota: 1 unit
+      - Permissions: full-scans:create
```

Comment:
☝️ I love the permission requirements but... I'll try to add that to the error messages too.